app/models/service.js

```diff
@@ -16,4 +17,21 @@ export default Resource.extend({
       return intl.t('model.service.displayKind.generic');
     }
   }),
+
+  proxyEndpoints: computed('labels', function(){
+    const parts = []
+    const labels = get(this, 'labels');
+    if ( labels && labels['kubernetes.io/cluster-service'] === 'true' ) {
+      (get(this, 'ports') || []).forEach((port) => {
+        const linkEndpoint = `${location.protocol}//${location.host}/k8s/clusters/${get(this, 'scope.currentCluster.id')}/api/v1/namespaces/${get(this, 'namespaceId')}/services/${get(port, 'targetPort')}:${get(this, 'name')}:/proxy/`;
```

Comment:
I'm surprised linting passes with this, it probably just doesn't support parsing the backtick string interpolation expressions.  `location` looks like an undefined variable, always use `window.` for things that are intentionally referring to global window variables.

Also `${location.protocol}//${location.host}` is just `${window.location.origin}`